src/TRestRawSignalDaqIdTaggingProcess.cxx

```diff
@@ -0,0 +1,152 @@
+/// * **name**: Name for the range. 
+/// * **ids**: Range of daq IDs. 
+///
+/// Each range must be defined under the "tag" section. 
+/// Any number of daq ID ranges can be defined (e.g. as many "tag" sections as needed).
+/// Numbers are assigned for each range, from 1 upwards.
+///
+/// Example in rml file: 
+/// \code
+/// <addProcess type="TRestRawSignalDaqIdTaggingProcess" name="TREXsides" value="ON" observable="all" >
+///     <tag name="South" ids="(0,575)"/>
+///     <tag name="North" ids="(576,1151)"/>
+/// </addProcess>
+/// \endcode
+///
+///  ### Observables
+///
+/// * **DaqIdRanges**: Each digit corresponds a daq ID range activated in the event,
+/// ordered with increasing order.
+/// As an example, in previous rml there are 3 possible values for this observable:
+///     * 1: South daq ID range.
+///     * 2: North daq ID range.
+///     * 12: South and North daq ID ranges.
+///
+///_______________________________________________________________________________
+///
+/// RESTsoft - Software for Rare Event Searches with TPCs
+///
+/// History of developments:
+///
+/// 2022-September: First implementation of TRestRawSignalDaqIdTaggingProcess
+///                 Created from TRestRawSignalAnalysisProcess
+///
+/// \class      TRestRawSignalDaqIdTaggingProcess
+/// \author     David Díez Ibáñez
+///
+/// <hr>
+///
+
+#include "TRestRawSignalDaqIdTaggingProcess.h"
+
+using namespace std;
+
+ClassImp(TRestRawSignalDaqIdTaggingProcess);
+
+///////////////////////////////////////////////
+/// \brief Default constructor
+///
+TRestRawSignalDaqIdTaggingProcess::TRestRawSignalDaqIdTaggingProcess() { Initialize(); }
+
+///////////////////////////////////////////////
+/// \brief Default destructor
+///
+TRestRawSignalDaqIdTaggingProcess::~TRestRawSignalDaqIdTaggingProcess() {}
+
+///////////////////////////////////////////////
+/// \brief Function to initialize input/output event members and define the
+/// section name
+///
+void TRestRawSignalDaqIdTaggingProcess::Initialize() {
+    SetSectionName(this->ClassName());
+    SetLibraryVersion(LIBRARY_VERSION);
+
+    fSignalEvent = nullptr;
+}
+
+///////////////////////////////////////////////
+/// \brief Process initialization.
+///
+void TRestRawSignalDaqIdTaggingProcess::InitProcess() {
+    
+}
+
+///////////////////////////////////////////////
+/// \brief Process initialization.
+///
+void TRestRawSignalDaqIdTaggingProcess::InitFromConfigFile() {
+    // This line is to exploit the retrieval of parameter as it is done at any process
+    TRestEventProcess::InitFromConfigFile();
+
+    // This is the additional code required by the process to read tags
+    TiXmlElement* tagDefinition = GetElement("tag");
+    while (tagDefinition) {
+        fTagNames.push_back( GetFieldValue("name", tagDefinition) );
+        fIdRanges.push_back( Get2DVectorParameterWithUnits("ids", tagDefinition) );
+
+        tagDefinition = GetNextElement(tagDefinition);
+    }
+        
+    PrintMetadata();

```

Comment:
```suggestion
    PrintMetadata();

```

Reply:
`PrintMetadata` is invoked externally during the processing chain when the process verboseLevel is set to `info`. Then, this `PrintMetadata` call will make that the process metadata is printed out several times. When instantiating it by other means, the user then needs to call `PrintMetadata` manually.